packages/peregrine/lib/talons/MagentoRoute/__tests__/useMagentoRoute.spec.js

```diff
@@ -51,7 +50,8 @@ const Component = props => {
 const routeComponentResults = {
     COMPONENT_FOUND: {
         component: {},
-        id: 1
+        id: 1,
+        store: 'default'
```

Comment:
Could also use jest global `STORE_VIEW_CODE`

https://github.com/magento/pwa-studio/pull/2643/files#diff-2d0cd5d10b9604941c38c6aac608178aR194

Reply:
This test doesn't need to access the global, as it's just a static string that needs to match up for the test to resolve correctly.